HighwayPatrol.cpp

```diff
@@ -6,11 +6,16 @@ HighwayPatrol::HighwayPatrol() : Vehicle("HighwayPatrol")
 
 }
 
+HighwayPatrol::~HighwayPatrol()
+{
+    
+}
+
```

Comment:
you can `= default;` this instead of providing an empty implementation.

Reply:
would not work with =default in header file got a vtable warning, no out of line virtual methods..